src/CLoop.cpp

```diff
@@ -120,3 +146,10 @@ void CLoop::Loop(float lumFactor, int z_sample, std::string key, const CLoopConf
     double time_spent = (endTime - startTime) / CLOCKS_PER_SEC;
     std::cout << "Time processing == " <<time_spent << std::endl;
 }
+
+int CLoop::GetNEntries()
+{
+  Long64_t nentries = fChain->GetEntries();
+  int n = static_cast<int>(nentries);
```

Comment:
This I think is quite dangerous. Casting something with more capacity to less capacity.